components/create_post/create_post.jsx

```diff
@@ -659,8 +664,7 @@ export default class CreatePost extends React.Component {
     }
 
     render() {
-        const stats = this.props.currentChannelStats;
-        const members = stats ? stats.member_count - 1 : '-';
+        const members = this.props.currentChannelMembersCount;
```

Comment:
Should i subtract -1 here? this is used for notifying user for @all with member count

Reply:
You're right.

Reply:
Done. BTW you guys should probably change build to trigger on latest commit and cancel the running build.